assets/common/common.go

```diff
@@ -184,7 +184,7 @@ func PostToSlack(path string, data url.Values) (SlackResponse, error) {
 	}
 	responseString := string(responseBytes)
 	if resp.StatusCode != 200 {
-		fmt.Printf(responseString)
+		fmt.Fprintf(os.Stderr, responseString)
```

Comment:
why not have just this as part of the error string on teh next line, or is it too large?